src/components/Splash.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div class="flex flex-col justify-center" @wheel="preventScrolling">
+    <BasicIcon
+      name="splash"
+      iconClass="w-11/12 bp-500:w-9/12 md:w-6/12 lg:w-5/12 mt-24 sm:mt-16 place-self-center"
+    />
+
+    <div
+      class="bg-primary flex flex-col space-y-16 bp-360:space-y-14 bp-420:space-y-10 lg:space-y-12 items-center rounded-2xl py-12 bp-500:py-10 md:py-11 lg:py-12"
+    >
+      <!-- title -->
+      <p class="font-londrina text-white text-5xl">{{ displayTitle }}</p>
+
+      <!-- metadata -->
+      <div class="flex flex-col space-y-4 w-full items-center">
+        <div :class="metadataContainerClass">
+          <div :class="metadataCellClass" class="border-r-2">
+            <BasicIcon
```

Comment:
`BaseIcon` may be a better name than `BasicIcon` since -- if I understand this correctly -- you're basiaclly extending the base icon to show different styles of icons

Reply:
I could simply rename it to `Icon` too for simplicity.